static/js/sefaria/sefaria.js

```diff
@@ -2254,6 +2254,29 @@ _media: {},
     }
     Sefaria.last_place = history_item_array.filter(x=>!x.secondary).concat(Sefaria.last_place);  // while technically we should remove dup. books, this list is only used on client
   },
+    isNewVisitor: () => {
```

Comment:
1. A new visitor is someone visiting Sefaria for the first time (which will be everyone for now) and they will be considered that until they close their tab/window and end their session. A returning visitor is anyone else who comes back after that initial time. (This was the original functionality that was intended)
2. Yep, that's what I intended. Unless they're logged in, I thought it was ok for everyone to be considered a new visitor once this code is released, but they will only be considered one for that first session. 
3. If this is truly an issue, I can make a true determination of whether someone is a new visitor or not by the existing cookies or session data. My goal was to not depend on any of the existing code that I removed. This is not a big problem unless the first modal or banner deployed after the release of code is intended for new visitors. I don't think it will be. There isn't a current goal to have a welcome message for new users/visitors either.
4. It isn't a real problem if a user can't be classified as a new or returning visitor, since it only potentially affects modals and banners right now. Local and session storage has been supported for a long time in browsers, so a fallback is not necessary. If local storage is going to be used for other data and my new utility functions are used in other places, then a fallback is necessary for when the storage limit is reached.